core/src/main/java/com/sequenceiq/cloudbreak/repository/StackRepository.java

```diff
@@ -207,6 +207,10 @@
     @Query("SELECT s.id FROM Stack s WHERE s.name= :name AND s.workspace.id= :workspaceId AND s.terminated = null")
     Optional<Long> findIdByNameAndWorkspaceId(@Param("name") String name, @Param("workspaceId") Long workspaceId);
 
+    @DisableCheckPermissions
+    @Query("SELECT s.id FROM Stack s WHERE s.resourceCrn= :crn AND s.workspace.id= :workspaceId AND s.terminated = null")
+    Optional<Long> findIdByCrnAndWorkspaceId(@Param("crn") String name, @Param("workspaceId") Long workspaceId);
```

Comment:
Please rename the param `name` to `crn`.

Reply:
Thanks, a nice catch! Done.